### PR TITLE
Add cli parser for multi-asset minting and policyid cli command

### PR DESCRIPTION
**Pull request by Jimbo4350**

### CODE REVIEW

File: cardano-cli/src/Cardano/CLI/Shelley/Parsers.hs [hunk 1]
```diff
@@ -1457,6 +1459,24 @@ pTxOut =
...
+    <> Opt.value ""
+    )
+
+pMintingScript :: Parser String
+pMintingScript =
+  Opt.strOption
+    (  Opt.long "minted-policy"
+    <> Opt.metavar "FILE"
+    <> Opt.help "Script containing the monetary policy for the minted multi-asset value(s)"
+    <> Opt.value ""
```
Comment (intricate) [inline cardano-cli/src/Cardano/CLI/Shelley/Parsers.hs:1477]:
```suggestion
    <> Opt.value ""
    <> Opt.completer (Opt.bashCompleter "file")
```

File: cardano-cli/src/Cardano/CLI/Shelley/Run/Transaction.hs [hunk 2]
```diff
@@ -105,7 +105,7 @@ renderShelleyTxCmdError err =
 runTransactionCmd :: TransactionCmd -> ExceptT ShelleyTxCmdError IO ()
 runTransactionCmd cmd =
   case cmd of
-    TxBuildRaw txins txouts ttl fee certs wdrls
+    TxBuildRaw txins txouts _Values _mPolicy ttl fee certs wdrls
```
Comment (intricate) [inline cardano-cli/src/Cardano/CLI/Shelley/Run/Transaction.hs:108]:
```suggestion
    TxBuildRaw txins txouts _mValues _mPolicy ttl fee certs wdrls
```

**Review by dcoutts — APPROVED** [review]
LGTM!

**Comment by dcoutts** [comment]
bors merge

**Pull request merged**